crates/sui-tool/src/commands.rs

```diff
@@ -386,6 +411,84 @@ impl ToolCommand {
                 let config = sui_config::NodeConfig::load(config_path)?;
                 restore_from_db_checkpoint(&config, &db_checkpoint_path).await?;
             }
+            ToolCommand::DownloadDBSnapshot {
+                epoch,
+                network,
+                genesis,
+                path,
+                skip_checkpoints,
+                skip_indexes,
+                num_parallel_downloads,
+            } => {
+                let snapshot_store_config = if network == Network::Mainnet {
+                    ObjectStoreConfig {
+                        object_store: Some(ObjectStoreType::S3),
+                        bucket: Some("mysten-mainnet-snapshots".to_string()),
```

Comment:
So it seems you made the decision in this PR to download mysten snapshots from the mysten hosted object stores, without support for downloading from other stores. This definitely makes it far easier to use the tool (the download side of my snapshot restore PR is a bit unweildy due to trying to provide this support). Just curious if there is a reason in your mind, or if you plan on extending later?

Reply:
yeah good call, i'll try to add bucket name and store type in the parameter list here